classes/wordpress.php

```diff
@@ -26,16 +27,18 @@ class Object_Sync_Sf_WordPress {
 	 * @param object $wpdb A wpdb object.
 	 * @param string $version The plugin version.
 	 * @param string $slug The plugin slug.
+	 * @param string $option_prefix The plugin's option prefix
 	 * @param object $mappings Mapping objects.
 	 * @param object $logging a Object_Sync_Sf_Logging instance.
 	 * @throws \Exception
 	 */
-	public function __construct( $wpdb, $version, $slug, $mappings, $logging ) {
-		$this->wpdb     = $wpdb;
-		$this->version  = $version;
-		$this->slug     = $slug;
-		$this->mappings = $mappings;
-		$this->logging  = $logging;
+	public function __construct( $wpdb, $version, $slug, $option_prefix, $mappings, $logging ) {
```

Comment:
Since __construct is a public method and this is the 1.4 release (not a 2.0 backwards compatibility breaking release), the new parameters should be optional. We can't guarantee developers haven't invoked this method.